docs/unpackaged.rst

```diff
@@ -1,14 +1,212 @@
-Manage unpackaged configuration
+Manage Unpackaged Configuration
 ===============================
 
-Deploy metadata after installing a package
-------------------------------------------
+Not everything that's part of an application can be part of a package.
 
-Namespace injection
+CumulusCI implements the Product Delivery Model by offering support for complex applications -- applications that may include multiple managed packages as well as unpackaged metadata, and setup automation that configures org settings or makes precise changes to existing configuration.
+
+The tools used to implement that support are *unpackaged metadata* and *Metadata ETL*. 
+
+Unpackaged metadata refers to metadata that is not delivered as part of a package, and can include both support metadata delivered to users as well as metadata that operationally configures orgs used by the product. 
+
+Metadata ETL is a suite of tasks that support surgically altering existing metadata in an org. It's a powerful technique that changes the unpackaged configuration in an org without risking damage by overwriting existing customizations with stored metadata. Metadata ETL is relevant for delivering applications to customers safely, and is often a superior alternative to unpackaged metadata.
+
+To learn more, see `Metadata ETL<TODO>`_.
+
+
+
+Roles of Unpackaged Metadata
+----------------------------
+
+
+``unpackaged/pre``: Prepare an Org
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some projects require that unpackaged metadata be deployed to finish the customization of an org *before* the project's own code and metadata are deployed.
+
+    Example: The Nonprofit Success Pack (NPSP) must deploy unpackaged Record Types prior to installing its own packages. ``unpackaged/pre`` is the location designed for such metadata, which is stored in subdirectories such as ``unpackaged/pre/first``.
+
+CumulusCI's standard flows that build orgs, such as ``dev_org`` and ``install_prod``, always deploy metadata bundles found in ``unpackaged/pre`` before proceeding to the deployment of the application. It's also easy to include ``unpackaged/pre`` metadata in customer-facing installers run via MetaDeploy.
+
+The ``deploy_pre`` task, which is part of the ``dependencies`` flow, is responsible for deploying these bundles.
+
+.. important:: Do not include metadata in ``unpackaged/pre`` unless it is intended to be delivered to *all* installations of the product.
+
+
+``unpackaged/post``: Configuration After Package Install
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Projects often include metadata that is genuinely part of the application, but either cannot be delivered as part of a managed package for operational reasons. This metadata must be deployed *after* the project's own code and metadata are deployed first and the org is configured.
```

Comment:
Changed up some of the wording in the first sentence and added the second one. Please confirm that everything is correct.

Reply:
```suggestion
Projects often include metadata that is genuinely part of the application, but either cannot be delivered as part of a managed package for operational reasons. This metadata must be deployed *after* the package's own code and metadata are deployed first and the org is configured.
```